### PR TITLE
Add updateAccount method

### PR DESCRIPTION
**Pull request by aminalaee**
Add updateAccount method according to the docs.

### CODE REVIEW

File: test/grpc/account_test.dart [hunk 1]
```diff
@@ -41,5 +41,9 @@ void main() {
       expect(users, isA<api.Users>());
       expect(users.users, hasLength(1));
     });
+
+    test('updating my account', () async {
+      await client.updateAccount(session: session, displayName: 'name');
+    });
```
Comment (aminalaee) [inline test/grpc/account_test.dart:47]: I'd expect to test a getAccount right after this, but seems like Nakama server updates account async so it's updated with a delay.

File: lib/src/nakama_client/nakama_grpc_client.dart [hunk 2]
```diff
@@ -299,6 +299,30 @@ class NakamaGrpcClient extends NakamaBaseClient {
...
+    await _client.updateAccount(
+      UpdateAccountRequest(
+        username: username == null ? null : StringValue(value: username),
+        displayName:
+            displayName == null ? null : StringValue(value: displayName),
+        avatarUrl: avatarUrl == null ? null : StringValue(value: avatarUrl),
+        langTag: langTag == null ? null : StringValue(value: langTag),
+        location: location == null ? null : StringValue(value: location),
+        timezone: timezone == null ? null : StringValue(value: timezone),
+      ),
```
Comment (aminalaee) [inline lib/src/nakama_client/nakama_grpc_client.dart:321]: @obrunsmann Sorry if the questions are many 😄 
I don't know what is your approach here, in protobuf it expects `StringValue` but I didn't see any usage of it, should I remove compiled code to switch from StringValue to String or keep it this way?

Comment (obrunsmann) [reply lib/src/nakama_client/nakama_grpc_client.dart:321]: No worries, just fire every question you have :) Again sorry for my delay here.

> should I edit compiled code
So **never** edit compiled code, I haven't checked this honestly speaking but I think there is no need to care about this...

Comment (aminalaee) [reply lib/src/nakama_client/nakama_grpc_client.dart:321]: Perfect, thanks.

**Pull request merged**